sql/moz-fx-data-shared-prod/fenix_derived/new_profile_activation_v1/query.sql

```diff
@@ -0,0 +1,123 @@
+WITH client_first_seen AS (
+  SELECT
+    client_id,
+    sample_id,
+    normalized_channel,
+    app_build,
+    app_channel,
+    app_display_version,
+    device_manufacturer,
+    device_model,
+    isp,
+    locale,
+    city,
+    country,
+    MIN(first_seen_date) AS first_seen_date
```

Comment:
Is this needed? Are there sometimes multiple values for `first_seen_date` for a given `client_id`+`submission_date`?

Reply:
I'll remove. We have duplicates, but they are across different days, so now that I'm making this incremental I think I'll need to de-dup in the final query that will use this table.